cli/utils.py

```diff
@@ -223,6 +224,84 @@ def print_batch_exception(batch_exception):
     log.error("-------------------------------------------")
 
 
+'''
+    Job submission
+'''
+
+def print_jobs(jobs: List[aztk.spark.models.Job]):
+    print_format = '{:<34}| {:<10}| {:<20}'
+    print_format_underline = '{:-<34}|{:-<11}|{:-<21}'
+
+    log.info(print_format.format('Job', 'State', 'Creation Time'))
+    log.info(print_format_underline.format('', '', '', ''))
+    for job in jobs:
+
+        log.info(
+            print_format.format(
+                job.id,
+                job.state,
+                utc_to_local(job.creation_time)
+            )
+        )
+
+def utc_to_local(utc_dt):
+    return utc_dt.replace(tzinfo=datetime.timezone.utc).astimezone(tz=None).strftime("%H:%M%p %d/%m/%y")
```

Comment:
nitpick: this method shouldn't be nested between job methods

nitpick2: Remove commented lines in print_job method